apps/ui/pages/reservation-unit/[id].tsx

```diff
@@ -639,23 +642,32 @@ const ReservationUnit = ({
         return false;
       }
 
-      if (
-        !isReservationReservable({
-          reservationUnit,
-          activeApplicationRounds,
-          start,
-          end,
-          skipLengthCheck: false,
-        })
-      ) {
+      // the next check is going to systematically fail unless the times are at least minReservationDuration apart
+      const { minReservationDuration } = reservationUnit;
+      let newEnd = end;
+      if (minReservationDuration) {
+        if (differenceInSeconds(end, start) < minReservationDuration) {
+          newEnd = addSeconds(start, minReservationDuration);
+        }
+      }
```

Comment:
Slightly cleaner way:
```
let newEnd = end
if (minReservationDuration) {
    newEnd = Math.max(end, addSeconds(start, minReservationDuration))
}
```
(we just added new linter rules to backend for similar cases :D)